docs/src/test-fixtures.md

```diff
@@ -335,3 +335,55 @@ const test = base.extend<{}, ExpressWorkerFixtures>({
 
 export default test;
 ```
+
+## Overriding fixtures
+
+In addition to creating your own fixtures, you can also override existing fixtures to fit your needs. Consider the following example which overrides the `page` fixture by navigating to a specified URL:
+
+```js js-flavor=js
+const base = require('@playwright/test');
+
+const test = base.test.extend({
+  baseURL: 'http://localhost:8080',
+  page: async ({ baseURL, page }, use) => {
+    await page.goto(baseURL);
+    await use(page);
+  },
+});
+```
+
+```js js-flavor=ts
+import { test as base } from '@playwright/test';
+
+const test = base.extend({
+  baseURL: 'http://localhost:8080',
+  page: async ({ baseURL, page }, use) => {
+    await page.goto(baseURL);
+    await use(page);
+  },
+});
+```
+
+Notice that in this example we added another fixture named `baseURL` which the `page` fixture uses. This allows us to override the `baseURL` used by the `page` fixture in our tests using `test.use`.
+
+```js
+test.use({ baseURL: 'https://playwright.dev' })
+```
+
+Fixtures can also be overridden where the base fixture is completely replaced with something different. For example, we could override the `storageState` fixture to provide our own data.
+
+```js js-flavor=js
+const base = require('@playwright/test');
+
+const test = base.test.extend({
+  storageState: { path: 'state.json' },
```

Comment:
@dgozman @mxschmitt I'm not really familiar with `storageState` or what @dgozman was suggesting in his comment, so if this is a poor example, let me know how it should be improved.

Reply:
I meant something like this:

```js
const test = base.test.extend({
  storageState: async ({}, use) => {
    const cookie = await getAuthCookie();
    await use({ cookies: [cookie] });
  }
});
```